.github/workflows/ci.yaml

```diff
@@ -33,16 +33,30 @@ jobs:
       RAILS_ENV: test
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      
+      - name: Set NodeJS Up
+        uses: actions/setup-node@v2
         with:
           node-version: 16
           cache: npm
-      - uses: ruby/setup-ruby@v1
+      
+      - name: Set Ruby Up
+        uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
       - run: npm install
       - run: make check
       - run: make lint
       - run: make test
       - run: make build
+
+      - name: Perform Configuration Files Scan
```

Comment:
Even here, I’m not sure about using a custom `name:`. I think the leaner the workflow, the better!